volatility3/framework/plugins/regexscan.py

```diff
@@ -46,38 +46,63 @@ def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]
             ),
         ]
 
-    def _generator(self, regex_pattern):
-        regex_pattern = bytes(regex_pattern, "UTF-8")
-        vollog.debug(f"RegEx Pattern: {regex_pattern}")
-
+    def _generator(self, compiled_pattern, raw_pattern, maxsize):
```

Comment:
The compiled pattern can be generated from the raw_pattern.  Asking for them both as parameters could lead to the two diverging (technically) and that breaking down the line.  Better to do the conversion to a compile pattern here to know that the value matches and won't change.